src/device/pf_fspm.c

```diff
@@ -1255,6 +1336,12 @@ int pf_fspm_signal_led_ind (pnet_t * net, bool led_state)
    int ret = 0;
    if (net->fspm_cfg.signal_led_cb != NULL)
    {
+      LOG_DEBUG (
```

Comment:
What is the intended use of the log levels? These are all log entries resulting from api calllabacks. Maybe they should be of higher log level info? I do not know, the application will problably create its own log entries for these callbacks anyway.

Reply:
I have found that when dissecting log files from users, it is very handy to know when the callbacks have been triggered. If callbacks are missing is also important information. This is mainly a debug feature, so I think the level is appropriate.